base_fontawesome/static/lib/fontawesome-5.3.1/css/brands.css

```diff
@@ -0,0 +1,13 @@
+/*!
```

Comment:
already mentioned in the old PR: do we really have to keep both compressed and uncompressed?
Our repos are already heavy... I think it's better to drop useless files especially if big :)

Reply:
I will remove all the unused files.